### PR TITLE
Nouiz mixed

### PR DESCRIPTION
**Pull request by abergeron**
Follow up of #3263 since @nouiz is unavailable for the next two weeks.

### CODE REVIEW

**Comment by abergeron** [comment]
Rebased (again).

File: doc/library/config.txt [hunk 1]
```diff
@@ -705,6 +705,24 @@ import theano and print the config variable, as in:
...
+
+    Bool value, default: True
+
+    Do NanGuardMode generate an error when it see an inf.
+
+.. attribute:: config.NanGuardMode.nan_is_error
+
+    Bool value, default: True
+
+    Do NanGuardMode generate an error when it see a big value (>1e10).
```
Comment (carriepl) [inline doc/library/config.txt]: For this attribute and the 2 previous, sentence structure is a bit weird. Replace "Do NanGuardMode [...]" with "Whether NanGuardMode should", maybe?

File: theano/compile/nanguardmode.py [hunk 2]
```diff
@@ -110,26 +128,60 @@ class NanGuardMode(Mode):
     big_is_error : bool
         If True, raise an error when a value greater than 1e10 is encountered.
 
+    Note
+    ----
+        We ignore the linker parameter
     """
+    # We currently loose the 3 first param freuquently, when calling
```
Comment (carriepl) [inline theano/compile/nanguardmode.py]: params
frequently

**Comment by carriepl** [comment]
Other than the two comment I made, this looks fine to me and @abergeron told me he already did quick review so once that's fixed and Travis is happy, it should be fine for merge

**Pull request merged**